### PR TITLE
Tree tooltip

### PR DESCRIPTION
**Pull request by albert-smit**
converted the text to a tooltip in the sunburst
a tooltip contains the distance, and if available a name, when the node is a ringnode  
or just a name when the node is a center node

### CODE REVIEW

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java [hunk 1]
```diff
@@ -52,27 +52,19 @@ public SunburstRingSegment(final PhylogeneticTreeItem value,
             final double scale) {
         // set the value, and create the text and semi-circle
         setValue(value);
-        setName(new Text(getValue().getName()));
+        String name = getValue().getName();
+        String tooltip;
+        if (name == null) {
+            tooltip = "Distance: " + getValue().getDistance();
```
Comment (jorenham) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstRingSegment.java]: duplicate code

File: lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstView.java [hunk 2]
```diff
@@ -148,7 +149,7 @@ private void update() {
     private void drawRingRecursive(final PhylogeneticTreeItem node,
             final int layer, final double degreeStart, final double degreeEnd) {
         // generate ring
-        SunburstRingSegment ringUnit = new SunburstRingSegment(node, layer,
+       SunburstRingSegment ringUnit = new SunburstRingSegment(node, layer,
```
Comment (jorenham) [inline lifetiles-tree/src/main/java/nl/tudelft/lifetiles/tree/view/SunburstView.java]: Indentation

**Comment by alangerak** [comment]
Looks good, if above is fixed

**Comment by rutgervandenberg** [comment]
Looks good aside from previously mentioned issues.

**Comment by joswinter** [comment]
Looks good.

**Comment by rutgervandenberg** [comment]
Theres still code duplication :\

**Pull request merged**